modules/monitoring-creating-cluster-id-labels-for-metrics.adoc

```diff
@@ -12,6 +12,14 @@ You can create cluster ID labels for metrics for default platform monitoring and
 For default platform monitoring, you add cluster ID labels for metrics in the `write_relabel` settings for remote write storage in the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace.
 
 For user workload monitoring, you edit the settings in the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace.
+
+[NOTE]
+====
+By design, the user workload monitoring Prometheus resource has the value of the `overrideHonorLabels` field set to `true`. 
```

Comment:
This is mostly implementation details. I'm not sure that it brings value to the reader.

Reply:
Deleted